uperf_wrapper/trigger_uperf.py

```diff
@@ -22,8 +22,8 @@
 
 class Trigger_uperf():
     def __init__(self, args):
-        self.uuid = args.uuid
-        self.user = args.user
+        self.uuid = args.uuid[0]
```

Comment:
why?  didn't realize there is more than 1 uuid or user involved.

Reply:
I looked at other wrappers and they don' t do this.

Reply:
I think the rebase is not completed yet